### ISSUE TITLE
Define how child works should be handled in a workflow.

### ISSUE THREAD

**Issue by jcoyne** [samvera/hyrax#1213]
If a work goes through a mediated workflow should permissions (remove edit) cascade to child works?

**Comment by elrayle** [comment]
When is this an issue?  I'm guessing...
* when the child work already exists and is added to a new work on the relationships tab
* when an existing work is brought back into the workflow (Can we do this?  I know there was discussion about being able to do this.

**Comment by jcoyne** [comment]
@elrayle yes.  Further complicating this issue is that the child work may have a different admin set.  Maybe we should prevent that from happening?

**Comment by mjgiarlo** [comment]
@jcoyne That sounds like a -tech list/call discussion.
